src/Parser/Priest/Discipline/Modules/Features/AtonementHealingBreakdown.js

```diff
@@ -82,9 +82,42 @@ class AtonementHealingBreakdown extends React.Component {
                     />
                   </td>
                 </tr>
-              );
-            })}
-        </tbody>
+
+              , (bolts && bolts.map((value,index) => {
+
+                const penanceBarPercentage = value / healing.effective;
+
+                if(!value) return null;
+
+                return (<tr>
+                  <td style={{ width: '30%', paddingLeft:50 }}>
+                    <SpellLink id={ability.guid}>
+                      <Icon icon={ability.abilityIcon} />{' '}
+                      {ability.name} Bolt {index + 1}
+                    </SpellLink>
+                  </td>
+                  {!this.state.absolute && (
+                  <td style={{ width: 60, paddingRight: 5, textAlign: 'center' }}>
+                    {(Math.round(value / totalAtonement.effective * 10000) / 100).toFixed(2)}%
```

Comment:
How about replacing `totalAtonement.effective` with `applicableTotal` which is defined above the return statement that takes care of the `this.state.absolute` toggle (on = its value is total, off = atonement relative)? I think this would reduce the amount of duplicate code.